### PR TITLE
fixed link id

### PR DESCRIPTION
**Pull request by tfitzmac**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/6117
This is an **experimental preview** for @microsoft users.

File: src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py [hunk 1]
```diff
@@ -609,7 +609,7 @@
 helps['resource link create'] = """
     type: command
     short-summary: Create a new link between resources.
-    long-summary: A link-id is of the form /subscriptions/{SubID}/resourceGroups/{ResourceGroupID}/{ProviderNamespace}/{ResourceType}/{ResourceName}/Microsoft.Resources/links/{LinkName}
+    long-summary: A link-id is of the form /subscriptions/{SubID}/resourceGroups/{ResourceGroupID}/providers/{ProviderNamespace}/{ResourceType}/{ResourceName}/providers/Microsoft.Resources/links/{LinkName}
```
Comment (vivsriaus) [inline src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py]: /providers is missing for other resource link commands below as well: resource link show, resource link update, resource link delete. Can you please fix them as well? Thanks.

**Comment by troydai** [comment]
@yugangw-msft

**Comment by yugangw-msft** [comment]
Please update the history doc to get CI green. There is a link above in the original comments which tells you how to do it.

**Comment by tfitzmac** [comment]
@yugangw-msft - it looks like I incorrectly checked that box in the original commit. This change only applies to help content, so I didn't think the history doc was needed.

**Comment by yugangw-msft** [comment]
@tfitzmac, I still think you need to upper the version in the setup.py since yours is the first one coming in since the release in last week

**Comment by yugangw-msft** [comment]
Please update history.rst with:
```
2.0.28
++++++
* Minor chanages
```

**Pull request merged**